extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -256,10 +257,19 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     const height = Number(this.element.getAttribute('height'));
     // Need to ensure these are numbers since width can be set to 'auto'.
     // Checking height just in case.
-    const useDefinedSizes = isExperimentOn(this.win, 'as-use-attr-for-format')
+    let useDefinedSizes = isExperimentOn(this.win, 'as-use-attr-for-format')
```

Comment:
Can we move from using isExperimentOn entirely and instead use src/experiments.js#randomlySelectUnsetExperiments?  See how a4a-safeframe-preloading-off is setup in doubleclick impl.  It will then handle the random and allow you to specify the experiment selection predicate (!response w/ height & width)

Reply:
Done.